### ISSUE TITLE
Selenium::WebDriver::Error::ElementClickInterceptedError:

### ISSUE THREAD

**Issue by aaronskiba** [portagenetwork/roadmap#503]
When executing the rSpec features/ tests, several GitHub Actions intermittently fail with the error `Selenium::WebDriver::Error::ElementClickInterceptedError:`

Because this error is intermittent, re-running the tests can solve the problem. However, this can also be quite time consuming (e.g. [a GitHub Action that had to be run 5 times to pass](https://github.com/portagenetwork/roadmap/actions/runs/6578720513/job/17903848540?pr=497)).

I've noticed this error when running the rSpec tests locally as well. The error doesn't seem to appear when I disable _headless_ mode. Perhaps the default window size in headless mode is the problem? If so, we could resolve this by specifying a window size argument.

**Comment by aaronskiba** [comment]
**Code Change (Below) Reveals the default window size is 800x600**
![Screenshot from 2024-02-07 15-02-30](https://github.com/portagenetwork/roadmap/assets/71047780/e51d1ef9-90bc-471f-81d5-97da12f94c05)
**Terminal Output**
```ruby
Randomized with seed 65255
Current window size: [800, 600]
FCurrent window size: [800, 600]
FCurrent window size: [800, 600]
```

**Comment by aaronskiba** [comment]
https://github.com/teamcapybara/capybara/blob/master/lib/capybara/registrations/drivers.rb includes the following code for the `:selenium_chrome_headless` driver:

```ruby
Capybara.register_driver :selenium_chrome_headless do |app|
  version = Capybara::Selenium::Driver.load_selenium
  options_key = Capybara::Selenium::Driver::CAPS_VERSION.satisfied_by?(version) ? :capabilities : :options
  browser_options = Selenium::WebDriver::Chrome::Options.new.tap do |opts|
    opts.add_argument('--headless=new')
    opts.add_argument('--disable-gpu') if Gem.win_platform?
    # Workaround https://bugs.chromium.org/p/chromedriver/issues/detail?id=2650&q=load&sort=-id&colspec=ID%20Status%20Pri%20Owner%20Summary
    opts.add_argument('--disable-site-isolation-trials')
  end

  Capybara::Selenium::Driver.new(app, **{ :browser => :chrome, options_key => browser_options })
end
```

We will have to add a `--window-size` arg to enable the selenium tests to work in DMP Assistant.

**Comment by aaronskiba** [comment]
A fix for this has been implemented within PR #700 (soon to be merged).
